systemFlake.nix

```diff
@@ -5,14 +5,17 @@
 , supportedSystems ? flake-utils-plus.lib.defaultSystems
 , inputs
 
-, nixosConfigurations ? { }
 , sharedExtraArgs ? { }
-, nixosProfiles ? { }
 , channels ? { }
 , channelsConfig ? { }
 , sharedModules ? [ ]
 , sharedOverlays ? [ ]
 
+, nixosConfigurations ? { }
+  # nixosProfiles should be deprecated since profiles aren't nixos specific
+, nixosProfiles ? { }
+, configProfiles ? nixosProfiles
```

Comment:
Not a big fan of the name, maybe `hosts` or something like that should be considered.

Reply:
I like hosts, but I'll probably submit the name change to a separate PR, as to discuss each change on its own.